packages/universal-components/src/Button/Button.stories.js

```diff
@@ -62,8 +63,11 @@ storiesOf('Button', module)
     }
     const circled = boolean('circled', false);
     const label = text('label', 'Playground button');
+    const style = object('style', {});
```

Comment:
I think we can remove style object from Storybook, it doesn't seems to work (or I'm just trying values in wrong format 😄). We don't have it there for any other component.

Reply:
Yeah this one is tricky, on web it works like `{"borderRadius": 10}` but didnt manage to make it work on mobile

Reply:
Ok, so let just remove it. As I mentioned it's not available for any other styleable component.

Reply:
Ok, removed!

Reply:
Thanks